internal/ingress/annotations/annotations_test.go

```diff
@@ -535,3 +535,36 @@ func TestExtractMethods(t *testing.T) {
 		})
 	}
 }
+
+func TestExtractSNIs(t *testing.T) {
+	type args struct {
+		anns map[string]string
+	}
+	tests := []struct {
+		name string
+		args args
+		want []string
+	}{
+		{
+			name: "empty",
+			want: []string{},
+		},
+		{
+			name: "non-empty",
+			args: args{
+				anns: map[string]string{
+					"konghq.com/snis": "hrodna.kong.example,katowice.kong.example",
```

Comment:
:grin: style improvement :grin:

```suggestion
					"konghq.com/snis": "hrodna.kong.example,wroclaw.kong.example",
```

_(joke, ignore this comment)_